client/src/components/LinkField/LinkField.js

```diff
@@ -164,6 +164,7 @@ const LinkField = ({
         description={data[linkID]?.description}
         versionState={data[linkID]?.versionState}
         typeTitle={type.title || ''}
+        typeIcon={type.icon || 'font-icon-link'}
```

Comment:
I don't think this `||` condition needs to be in here, since the icon will fall back to `font-icon-link` as defined on the `Link` class anyway.
```suggestion
        typeIcon={type.icon}
```

Reply:
Updated